src/native/libs/System.Native/pal_time.c

```diff
@@ -53,6 +53,33 @@ int32_t SystemNative_UTimensat(const char* path, TimeSpec* times)
     return result;
 }
 
+int32_t SystemNative_FUTimens(intptr_t fd, TimeSpec* times)
+{
+    int32_t result;
+
+#if HAVE_FUTIMENS
```

Comment:
doesn't have to happen in this PR, but perhaps this code and that of https://github.com/dotnet/runtime/blob/4019e83878a81465f6e42e8502b53bc5d1752f81/src/native/libs/System.Native/pal_io.c#L1311-L1326 should be shared.

Reply:
@tmds Is something like this already planned? In your opinion, should this be changed in this PR or separately?

Reply:
I'm not sure the result would be an improvement because a common function would probably use `TimeSpec` and that means we'd need to add conversions when calling it from `SystemNative_CopyFile`.

Let's keep it out-of-scope for this PR.

Reply:
Agreed. Just spotted the duplication. Let's leave it